### DISCUSSION TITLE
An error occurred loading the site extensions information

### DISCUSSION THREAD

**Discussion by Martin511** [akeeba/panopticon#690] [Q&A (Community Support)] [answered]
Hello. 
On one site (and this is the only site that is on Joomla 4.4.5, all others are on Joomla 5.1.1) I get an error regarding the check for extension updates. 
Error message: 
"An error occurred loading the site extensions information. 
cURL error 28: Operation timed out after 30001 milliseconds with 0 bytes received (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://www.xxxxxxxxxxxxxxxxx.de/api/index.php/v1/panopticon/extensions?page%5Blimit%5D=10000&force=1"

I tried to rename the htaccess temporarily and deactivated the firewall, no success. 
How can I find out why this happens?

**Comment by Martin511** [comment, marked as answer]
I have a solution, but I don´t know why:
Rebuilding the "Update Sites" in the Joomla site´s backend has solved it.

**Reply by nikosdion** [reply]
When you fetch the list of extensions, we ask Joomla for their update status. Each extension is linked to one or more update sites. Joomla stores the time and date it last checked for updates for this update site. If it's over the time limit configured in the System, Update, Updates Sites, Options page it will try to load the URL of the update site and fetch the update information from it.

You had at least one update site which was pointing to a URL which did not respond at all within 30 seconds. Therefore, Joomla reported an error.

The typical approach in this case is to disable that update site.

When you use Rebuild in Update Sites you **DELETE** all update sites. Then, you tell Joomla to enumerate (list) all extensions, and check the cached XML manifest of each extension for an update site. If one exists, it is added to the Update Sites page and linked to that extension.

While this is a solution, it's not always the best solution. Joomla tries not to lose Download Keys (which are part of each Update Site record in the database) but this will only work if the new update site is identical (including lower and uppercase characters) to the old one. Moreover, some extensions may list a generic or no update site, and rely on the post-installation code they ship with to provide the correct update site URL to Joomla. In short, Rebuild should be your absolute last resort.

**Comment by Martin511** [comment]
Thank you for the information.
But now it is too late, I have clicked the button.
It didn't seem critical to me, as there were no warnings. 
I hope that this has no negative consequences now...
